tests/e2e/e2e_upgrade_test.go

```diff
@@ -1,7 +1,6 @@
 package e2e
 
 import (
-	"os/exec"
```

Comment:
We can remove the `// Re-deploy policies` code in line 218 as a part of this PR too